src/pages/[platform]/build-a-backend/auth/v6-change-log/index.mdx

```diff
@@ -0,0 +1,147 @@
+
+// Deprecated: use fetchMFAPreference => output.enabled instead 
+-    Auth.getMFAOptions: (user: CognitoUser | any) => Promise<MFAOption[]>;
+
+-    Auth.getPreferredMFA: (user: CognitoUser | any, params?: GetPreferredMFAOpts) => Promise<string>;
++    fetchMFAPreference: () => Promise<FetchMFAPreferenceOutput>;
+
+-    Auth.setPreferredMFA: (user: CognitoUser | any, mfaMethod: 'TOTP' | 'SMS' | 'NOMFA' | 'SMS_MFA' | 'SOFTWARE_TOKEN_MFA') => Promise<string>;
++    updateMFAPreference: (input: UpdateMFAPreferenceInput) => Promise<void>;
+
+// Deprecated: use updateMFAPreference({sms: 'DISABLED'})
+-    Auth.disableSMS: (user: CognitoUser) => Promise<string>;
+     
+// Deprecated: use updateMFAPreference({sms: 'ENABLED'})
+-    Auth.enableSMS: (user: CognitoUser) => Promise<string>;
+
+-    Auth.setupTOTP: (user: CognitoUser | any) => Promise<string>;
++    setUpTOTP: () => Promise<SetUpTOTPOutput>;
+
+-    Auth.verifyTotpToken(user: CognitoUser | any, challengeAnswer: string) => Promise<CognitoUserSession>;
++    verifyTOTPSetup: (input: VerifyTOTPSetupInput) => Promise<void>;
+
+-    Auth.confirmSignIn: (user: CognitoUser | any, code: string, mfaType?: 'SMS_MFA' | 'SOFTWARE_TOKEN_MFA' | null, clientMetadata?: ClientMetaData) => Promise<CognitoUser | any>;
++    confirmSignIn: (input: ConfirmSignInInput) => Promise<ConfirmSignInOutput>;
+
+// Deprecated: use confirmSignIn({challengeResponse: 'newPassword'})
+-    Auth.completeNewPassword: (user: CognitoUser | any, password: string, requiredAttributes: any = {}, clientMetadata?: ClientMetaData) => Promise<CognitoUser | any>;
+
+// Deprecated: use confirmSignIn({challengeResponse: 'answer'})
+-    Auth.sendCustomChallengeAnswer: (user: CognitoUser | any, challengeResponses: string, clientMetadata?: ClientMetaData) => Promise<CognitoUser | any>;
+
+
+-    Auth.deleteUserAttributes: (user: CognitoUser | any, attributeNames: string[]) => Promise<any>;
++    deleteUserAttributes: (input: DeleteUserAttributesInput) => Promise<void>;
+
+-    Auth.deleteUser: () => Promise<string | void>;
++    deleteUser: () => Promise<void>;
+
+-    Auth.updateUserAttributes: (user: CognitoUser | any, attributes: object, clientMetadata?: ClientMetaData) => Promise<string>;
++    updateUserAttributes: (input: UpdateUserAttributesInput) => Promise<UpdateUserAttributesOutput>;
+
+-    Auth.userAttributes: (user: CognitoUser | any) => Promise<CognitoUserAttribute[]>;
++    fetchUserAttributes: () => Promise<FetchUserAttributesOutput>;
+
+// Deprecated: use fetchUserAttributes() output.email_verified and output.phone_number_verified
+-    Auth.verifiedContact: (user: CognitoUser | any) => Promise<any>;
+     
+-    Auth.currentUserPoolUser: (params?: CurrentUserOpts) => Promise<CognitoUser | any>;
++    getCurrentUser: () => Promise<GetCurrentUserOutput>;
+
+-    Auth.currentAuthenticatedUser: (params?: CurrentUserOpts) => Promise<CognitoUser | any>;
++    getCurrentUser: () => Promise<GetCurrentUserOutput>;
```

Comment:
`Auth.currentAuthenticatedUser` returned additional data that is not provided by `GetCurrentUserOutput`, for example session related information. Can we please ensure that **all** output changes are accounted for? Also to match the rest of these examples, `Auth.currentAuthenticatedUser` should be noted as deprecated right?